ChemicalHandler.py

```diff
@@ -16,15 +16,17 @@ class InputHandler():
     def __init__(self):
         self.input_dict = {}
          
-    def getReactants(self,reactant):
+    def getReactants(self,reactant) -> bool:

```

Comment:
please, add typing of input arguments as well as docstring to describe your function